includes/wf_crm_webform_postprocess.inc

```diff
@@ -1693,11 +1693,12 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
 
     // Only if #installments = 1, do we process a single transaction/contribution. #installments = 0 in CiviCRM Core means open ended commitment;
     $numInstallments = wf_crm_aval($contributionParams, 'installments', 1, TRUE);
-    if ($numInstallments == 1) {
-      $result = wf_civicrm_api('contribution', 'transact', $contributionParams);
+    $frequencyInterval = wf_crm_aval($contributionParams, 'frequency_unit', NULL, TRUE);
```

Comment:
This may be overkill. You're setting the `wf_crm_aval` function to use strict mode here but then using the not-so-strict `!empty()` check below. It's probably safe to leave off the last 2 params here.